packages/ra-ui-materialui/src/field/ReferenceOneField.tsx

```diff
@@ -66,6 +66,16 @@ export const ReferenceOneField = <
         link,
     });
 
+    const renderEmptyItemOption = useCallback(() => {
+        return typeof emptyText === 'string' ? (
+            <Typography component="span" variant="body2">
+                {emptyText && translate(emptyText, { _: emptyText })}
+            </Typography>
+        ) : emptyText ? (
+            emptyText
+        ) : null;
+    }, [emptyText, translate]);
```

Comment:
You don't need all this. Just add the `useCallback` function content directly where you currently call it